logging/opensearch/bin/run_securityadmin.sh

```diff
@@ -27,3 +27,5 @@ while [ $rc != 0 ] && [ $counter -lt 3 ]; do
   echo "RC: $rc"
   ((counter++))
 done
+
+exit $rc
```

Comment:
Interestingly, without this "exit" statement, the script would often end with a non-zero exit code (1) that appears to be triggered by the while loop.  It wasn't really causing up problems but the DAC folks have reported an intermittent problem caused by something in our code exiting with a non-zero exit code.  But there was no messages indicating what/where the failure was happening.  Not sure if this will fix it, but it at least ensures our script exits with an rc that accurately reflects the status of the command we executed.